explorer/ast/value.cpp

```diff
@@ -414,10 +414,14 @@ void Value::Print(llvm::raw_ostream& out) const {
     case Value::Kind::TupleValue: {
       out << "(";
       llvm::ListSeparator sep;
-      for (Nonnull<const Value*> element :
-           cast<TupleValueBase>(*this).elements()) {
+      const auto elements = cast<TupleValueBase>(*this).elements();
+      for (Nonnull<const Value*> element : elements) {
         out << sep << *element;
       }
+      // print trailing comma for single element tuples: (i32,)
```

Comment:
```suggestion
      // Print trailing comma for single element tuples: (i32,).
```

We use sentence structure for comments, including capitalization and punctuation.